igor-web/src/test/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverterTest.groovy

```diff
@@ -36,4 +36,22 @@ class TravisResultConverterTest extends Specification {
         "passed"     | Result.SUCCESS
         "errored"    | Result.FAILURE
     }
+
+    def "check if build is running"() {
+        setup:
+        def state = travisBuildState
+
+        when:
+        boolean result = TravisResultConverter.running(state)
+
+        then:
+        result == expectedResult
+
+        where:
+        travisBuildState | expectedResult
```

Comment:
Minor aesthetics but it's nice to line up the columns (IntelliJ reformat will do the alignment).

Also consistent to use `||` between inputs and expected results.

```
where:
travisBuildStage || expectedResult
"started"        || true
"created"        || true
```